heron/statemgrs/src/java/com/twitter/heron/statemgr/FileSystemStateManager.java

```diff
@@ -161,4 +180,52 @@ public void initialize(Config config) {
   private static String concatPath(String basePath, String appendPath) {
     return String.format("%s/%s", basePath, appendPath);
   }
+
```

Comment:
Since this is a nice tool - it might be good to separate this out in a separate file - so that it works for any state manager. We can make it work for any state manager?